src/components/dev-hub/theme.js

```diff
@@ -40,6 +40,7 @@ const size = {
     small: '10px',
     default: '16px',
     medium: '20px',
+    mediumLarge: '24px',
```

Comment:
Dan seems to be using this value more often, but I didn't want to replace `medium`.

Reply:
TIHI